AfterEffects/22.0/index.d.ts

```diff
@@ -1381,6 +1381,9 @@ declare class KeyframeEase {
 }
 
 declare class Layer extends PropertyGroup {
+  /** The unique and persistent identification number */
+  readonly id: number
```

Comment:
But this should still be optional, not to screw up earlier versions?

`readonly id?: number`

Reply:
@ivansenic I am not sure, it's a separate file `AfterEffects/22.0/index.d.ts`

Reply:
ah perfect then!